assets/src/styles/blocks.scss

```diff
@@ -39,19 +39,20 @@
 }
 
 .is-style-space-evenly {
+  gap: 24px !important;
```

Comment:
Just take in mind that all spacing classes aren't available at `blocks.scss`, since the `variables.scss` isn't be loaded.